events/views.py

```diff
@@ -63,8 +63,16 @@ def get_context_data(self, **kwargs):
     def get_template_names(self):
         event_title = self.get_context_data().get('event').title.lower()
         logger.debug(event_title)
-        if event_title in ['årsfest', 'årsfest gäster']:
+        if event_title in ['årsfest', 'årsfest gäster']: # TODO: Selectable template
             return ['events/arsfest.html']
+        elif event_title in ['100 baal']:
+            return ['events/kk100_detail.html']
+        elif event_title in ['baal']:
+            return ['events/baal_detail.html']
+        elif event_title in ['tomtejakt']:
+            return ['events/tomtejakt.html']
+        elif event_title in ['wappmiddag']:
+            return ['events/wappmiddag.html']
```

Comment:
Will these return a 500 if you create an event with that title on e.g. Date and template is not found?

Maybe add a template to common if it doesn't already exist